lab-regan/server.js

```diff
@@ -0,0 +1,73 @@
+'use strict';
+
+const express = require('express');
+const morgan = require('morgan');
+const createError = require('http-errors');
+const debug = require('debug')('neighbor:server');
+const jsonParser = require('body-parser').json();
+
+const PORT = process.env.PORT || 3000;
+const app = express();
+
+const storage = require('./lib/storage.js');
+const Neighbor = require('./model/neighbor.js');
+
+app.use(morgan('dev'));
+
+app.get('/test', function(req,res){
+  debug('GET: /test');
+  res.json({msg: 'test route works'});
+});//end get test
+
+app.get('/api/neighbor/', function(req,res, next){
+  debug('/api/neighbor/');
+  storage.fetchItem();
```

Comment:
I think maybe you should be calling your Neighbor.fetchNeighbor function here instead of testing the storage function.